geometry_msgs/PointStamped min
geometry_msgs/PointStamped max